client/src/EMS/RingdownForm.js

```diff
@@ -139,9 +139,10 @@ function RingdownForm({ defaultPayload, className }) {
               </p>
             </div>
           </div>
-          {step === 0 && <PatientFields onChange={onChange} ringdown={ringdown} />}
-          {step === 1 && <HospitalSelection onChange={onChange} ringdown={ringdown} />}
+         
+
           <fieldset className="usa-fieldset border-top border-base-lighter">
+            {step === 0 && <PatientFields onChange={onChange} ringdown={ringdown} />}
```

Comment:
Can you combine this with the next block, so there's just one `step === 0` check?  The `PatientFields` will probably need to go into the fragment.  Same with step 1.

Reply:
Can Do!